### PR TITLE
Added OTP verifier python script

### PR DESCRIPTION
**Pull request by Ash515**
# Description

Added OTP verifier python Script.

### CODE REVIEW

File: Python/OTP_verifier.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import random
+from twilio.rest import Client
+
+otp=random.randint(1000,9999)
+print(otp)
+account_sid='Find from your twilio account'
```
Comment (HarshCasper) [inline Python/OTP_verifier.py:6]: Keep this in a `.env` file and use a module like `dotenv` to import them in your Scipt. It makes it easy for us to track the Environment Variables.

File: Python/OTP_verifier.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import random
+from twilio.rest import Client
+
+otp=random.randint(1000,9999)
```
Comment (HarshCasper) [inline Python/OTP_verifier.py:4]: The Random Generatic Logic is not that secure and strong, as far as I understand. You can take a look at this [blog](https://realpython.com/python-random/) for better understanding.

File: Python/OTP_verifier.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+print(otp)
+account_sid='Find from your twilio account'
+auth_token='Finfd from your twilio account'
+client=Client(account_sid,auth_token)
+
+message=client.message.create(
```
Comment (HarshCasper) [inline Python/OTP_verifier.py:10]: The Code seems to be unformatted. Format your Code properly and indent it for better understanding.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please add a proper README as well with instructions on kick-starting the Script.

**Comment by HarshCasper** [comment]
Are you still working on the same?

**Comment by Ash515** [comment]
No I have practical examination so I not go to that repo. Tomorrow I will
get free and start to solve those problems

**Comment by HarshCasper** [comment]
Hello @Ash515 

I am closing this PR right now. If you are still looking to work on the same, feel free to ping me up and I will open it up 👍

**Pull request closed without merging**